package.json

```diff
@@ -4,15 +4,19 @@
   "description": "Webstudio Designer UI as a service",
   "scripts": {
     "build": "remix build",
-    "dev": "node -r dotenv/config node_modules/.bin/remix dev",
-    "postinstall": "remix setup node",
+    "dev": "yarn setup && node -r dotenv/config node_modules/.bin/remix dev",
+    "start": "remix-serve build",
     "typecheck": "tsc -b",
     "lint": "eslint . --ext .ts,.tsx --max-warnings 0",
     "test": "jest",
+    "setup": "cp ./node_modules/@webstudio-is/sdk/prisma/schema.prisma prisma/ && yarn migrate && yarn generate",
     "checks": "yarn lint && yarn typecheck && yarn test",
-    "watch:sdk": "relative-deps watch"
+    "migrate": "yarn prisma migrate dev",
```

Comment:
did it need yarn prisma or yarn relative-deps? I remember we tried that but I recall inside scripts it can be just the command

Reply:
oh unless the script is being called from npm run and so this woiuldn't run through yarn :)

Reply:
It does not you are right, removed!